cypress/integration/HeaderBar/The_HeaderBar_should_contain_a_logo_that_links_to_the_homepage/HeaderBar_contains_logo.js

```diff
@@ -1,14 +1,13 @@
 import '../common/index.js'
 import { Then } from 'cypress-cucumber-preprocessor/steps'
+import { baseUrl } from '../common/index.js'
```

Comment:
I guess the import in line 3 should be enough and we can remove the one in line 1